src/Cake.Core.Tests/Unit/Scripting/CodeGen/ParameterEmitterTests.cs

```diff
@@ -424,6 +428,7 @@ public static void OptionalNullableBoolWithNonNullDefault(bool? arg = true)
         [InlineData("OptionalNullableFloatWithNonNullDefault", "System.Nullable<System.Single> arg = (System.Single)1")]
         [InlineData("OptionalNullableEnumWithNonNullDefault", "System.Nullable<Cake.Core.Tests.Unit.Scripting.CodeGen.ParameterEmitterTests+TestEnum> arg = (Cake.Core.Tests.Unit.Scripting.CodeGen.ParameterEmitterTests+TestEnum)1")]
         [InlineData("OptionalNullableBoolWithNonNullDefault", "System.Nullable<System.Boolean> arg = (System.Boolean)true")]
+        [InlineData("ParameterUsingReservedKeyword", "System.Int32 @new")]
```

Comment:
Could we add some tests here (for reserved keywords) that tests:

* required value
* optional value
* required nullable value
* optional nullable value

```csharp
[InlineData("RequiredIntKeyword", "System.Int32 @new")]
[InlineData("OptionalIntKeyword", "System.Int32 @new = (System.Int32)1")]
[InlineData("RequiredNullableIntKeyword", "System.Nullable<System.Int32> @new")]
[InlineData("OptionalNullableIntKeywordWithNullDefault", "System.Nullable<System.Int32> @new = null")]
```
*